src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netcoreapp2.1</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <Nullable>disable</Nullable>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.5</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.0.1.2</AssemblyVersion><!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.1.2</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.2.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.5.5</PackageValidationBaselineVersion>
+    <IsPlaceholderTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp2.1'">true</IsPlaceholderTargetFramework>
+    <DefineConstants Condition="'$(IsPlaceholderTargetFramework)' != 'true'">$(DefineConstants);NETSTANDARD</DefineConstants>
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
```

Comment:
Does setting this property here have any effect in this repo? We had infra for that in corefx back then. I don't think we invoke the linker.

Reply:
Okay. I'll remove it. Hopefully something will fail if this was actually needed.

Reply:
@ericstj do you feel strongly about invoking the linker on our assembly to mimic this behavior? I can't remember if we productized clear init locals in the compiler.

Reply:
Yeah, the roslyn feature is called `SkipLocalsInit`: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/general#skiplocalsinit-attribute

Reply:
@carlossanlop you want to add this file to the Common folder and import it in this project: https://github.com/dotnet/runtime/blob/acc3f0cea4057f8bde7ffebdabd1dc62c4ebef0b/src/libraries/Common/src/SkipLocalsInit.cs#L3

That's how you achieve the same as `ILLinkClearInitLocals` but now with the compiler instead of the linker.

Reply:
@ericstj mentioned offline that we might not need to do this for the .NET Framework and .NET Standard builds.

Reply:
I'm still not sure why we want this enabled. Two reasons:
- I don't see any usages of `SkipLocalsInit` in System.Memory.
- If I add the SkipLocalsInit.cs file to Common per your suggestion, I'd have to also add SkipLocalsInitAttribute.cs, which comes from System.Private.CoreLib.

Reply:
Ah we wrote the messages at the same time. Okay, I won't add it. Thanks for checking.

Reply:
It was used for the .NETCoreApp build, which doesn't apply here anymore.